SteamBot/Bot.cs

```diff
@@ -336,6 +364,24 @@ public void HandleBotCommand(string command)
             }
         }
 
+        public void HandleInput(string input)
+        {
+            consoleInput = input;
+        }
+
+        public string WaitForInput()
+        {
+            consoleInput = null;
+            while (true)
+            {
+                if (consoleInput != null)
+                {
+                    return consoleInput;
+                }
+                Thread.Sleep(5);
+            }
+        }
```

Comment:
No different than this: https://github.com/Jessecar96/SteamBot/blob/master/SteamBot/Bot.cs#L942